<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomWindow</class>
 <widget class="QWidget" name="CustomWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>556</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>LifeCraft</family>
    <weight>75</weight>
    <bold>true</bold>
    <underline>false</underline>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="titleBar" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>23</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="LTitle">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbMin">
        <property name="minimumSize">
         <size>
          <width>29</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>29</width>
          <height>20</height>
         </size>
        </property>
        <property name="iconSize">
         <size>
          <width>29</width>
          <height>20</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbSmallWindow">
        <property name="minimumSize">
         <size>
          <width>27</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>27</width>
          <height>20</height>
         </size>
        </property>
        <property name="iconSize">
         <size>
          <width>26</width>
          <height>20</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbMax">
        <property name="minimumSize">
         <size>
          <width>27</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>27</width>
          <height>20</height>
         </size>
        </property>
        <property name="iconSize">
         <size>
          <width>26</width>
          <height>20</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbFullscreen">
        <property name="minimumSize">
         <size>
          <width>26</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>26</width>
          <height>20</height>
         </size>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbClose">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>20</height>
         </size>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>20</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="centralWidget" native="true">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="leftBorder" native="true">
        <property name="minimumSize">
         <size>
          <width>21</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>10</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="centralLeft" native="true">
        <property name="minimumSize">
         <size>
          <width>260</width>
          <height>500</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>260</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <widget class="QPushButton" name="btnOpenInExplorer">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>390</y>
           <width>134</width>
           <height>28</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
          </font>
         </property>
         <property name="text">
          <string>Open map in explorer</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnCreateNewDB">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>150</y>
           <width>134</width>
           <height>28</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
          </font>
         </property>
         <property name="text">
          <string>Create new database</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnResetFilters">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>190</y>
           <width>134</width>
           <height>28</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
          </font>
         </property>
         <property name="text">
          <string>Reset Filters</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnLoad">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>110</y>
           <width>134</width>
           <height>28</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
          </font>
         </property>
         <property name="text">
          <string>Load/Refresh maps</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnMoveMap">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>430</y>
           <width>241</width>
           <height>69</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
          </font>
         </property>
         <property name="text">
          <string>Move map to current map-path and move
all other maps in map-path to a backup
folder</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnSetMapDir">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>134</width>
           <height>28</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="text">
          <string>Choose map-path</string>
         </property>
         <property name="iconSize">
          <size>
           <width>200</width>
           <height>50</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>282</y>
           <width>78</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
          </font>
         </property>
         <property name="text">
          <string>Max players:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>338</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
          </font>
         </property>
         <property name="text">
          <string>File-name:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="leFilterIngameName">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>308</y>
           <width>154</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="lMapDir">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>109</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
          </font>
         </property>
         <property name="text">
          <string>Current map-path:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>226</y>
           <width>93</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
          </font>
         </property>
         <property name="text">
          <string>Minimum rating:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>366</y>
           <width>31</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
          </font>
         </property>
         <property name="text">
          <string>Tags:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="leFilterFileName">
         <property name="geometry">
          <rect>
           <x>79</x>
           <y>336</y>
           <width>175</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="leFilterTags">
         <property name="geometry">
          <rect>
           <x>49</x>
           <y>364</y>
           <width>205</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>254</y>
           <width>128</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
          </font>
         </property>
         <property name="text">
          <string>Minimum max players:</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="sbMinimumRating">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>224</y>
           <width>30</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="lMapDirStr">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>241</width>
           <height>45</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
          </font>
         </property>
         <property name="text">
          <string>&lt;none selected&gt;

</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="sbMaxPlayers">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>280</y>
           <width>40</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>310</y>
           <width>79</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
          </font>
         </property>
         <property name="text">
          <string>Ingame-name:</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="sbMinimumMaxPlayers">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>252</y>
           <width>40</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="centralRight" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTableView" name="view"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="rightBorder" native="true">
        <property name="minimumSize">
         <size>
          <width>21</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>10</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="bottomBorder" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>23</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>CustomWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
